backends/platform/ds/ds.mk

```diff
@@ -170,10 +171,10 @@ dsclean:
 
 # HACK/FIXME: C compiler, for cartreset.c -- we should switch this to use CXX
 # as soon as possible.
-CC := $(DEVKITPRO)/devkitARM/bin/arm-eabi-gcc
+CC := $(DEVKITPRO)/devkitARM/bin/arm-none-eabi-gcc
```

Comment:
This change broke buildbot because `arm-none-eabi-gcc` is not available. I guess we need to update our toolchain?